tools/ci_build/github/azure-pipelines/win-wasm-ci-pipeline.yml

```diff
@@ -39,6 +39,22 @@ jobs:
         BuildConfig: 'Release'
         CmdParams: '--enable_wasm_threads --skip_tests --disable_wasm_exception_catching --disable_rtti'
         WasmFileName: 'ort-wasm-threaded'
+      'simd debug':
```

Comment:
we now have a matrix-size=8 for job build_WASM. is this a concern for the pipeline?

Reply:
some possible way to save CI resources:
- use linux to build wasm (if linux is cheaper)
- an alternative solution is to use matrix size = 2 only for debug and release, inside each job we build 4 tasks in sequence.

any comments @snnn?

Reply:
Can we skip the debug builds? All the other pipelines disabled it for saving resources.

Reply:
we only run tests on debug build, because for release build, we disable exception catching (this causes onnxruntime_test_all to fail because it works depends on exceptions)

Reply:
goes the 2nd solution.